src/ray/protobuf/event.proto

```diff
@@ -35,8 +35,8 @@ message Event {
   string label = 6;
   // message is main information of this event
   string message = 7;
-  // timestamp is the report millionseconds since 00:00, Jan 1 1970 UTC
-  int64 timestamp = 8;
+  // timestamp is the format string like 2020-08-29 14:01:02.016101
+  string timestamp = 8;
```

Comment:
Can you explain that why string format is better than int64?

Reply:
Maybe string format is readable than int64? User doesn't have to transfer to a readable time structure

Reply:
I think the users of event framework (e.g., dashboard) can just take care of parsing timestamp (and we can just provide timestamp). Let's just use int64 timestamp (for generality)

Reply:
@Basasuya You may define two parameters including both string timestamp and timezone if we keep it. But in int64 time format, any component can convert it to other timezone.

Reply:
OK, I will change to the original defination.